dashboard/utils.py

```diff
@@ -102,7 +102,7 @@ def _get_program_fa_prices(self):
         """
         Returns the financial aid possible cost range.
         """
-        course_max_price = self.program.get_course_price()
+        course_max_price = self.program.price
```

Comment:
@noisecapella i just merged a PR that moves this code. on master this is now in `financialaid.serializers.FinancialAidDashboardSerializer#get_program_price_range`